### DISCUSSION TITLE
Maxpoints: how to edit variable in command line

### DISCUSSION THREAD

**Discussion by sarveshkoyande** [alicevision/Meshroom#1583] [Q&A]
Hi, I'm working on meshroom in Google Collab.
My project get stuck at meshing stage indefinitely.
I found a suggestion online that maxpoints reduction can resolve the issue.
So, I imported the cache upto depthmapfilter and ran the project in meshroom for windows and it solved the meshing issue with 750,000 max points. However, the texturing fails without giving any error. ( Uploaded the cache back to.gdrive but the texturing program now asks for file called densereconstrcution.bin which was never generated in the first place.

I think if the original issue of changing maxpoints in Google Collab is somehow resolved that would be easier. Can anybody please suggest how it can be done?

**Comment by natowi** [comment]
https://github.com/alicevision/meshroom/issues/243#issuecomment-997920471
and https://github.com/alicevision/meshroom/issues/851#issuecomment-727176535

**Reply by sarveshkoyande** [reply]
Thankyou very much for Swift response, I also figured out that I was using different version in Collab and on pc.
